opensrp-wcaro/src/main/java/org/smartgresiter/wcaro/fragment/AddMemberFragment.java

```diff
@@ -0,0 +1,285 @@
+
+    public void setContext(Context context){
+        this.context = context;
+    }
+    public void setFamilyBaseEntityId(String familyBaseEntityId){
+        this.familyBaseEntityId = familyBaseEntityId;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setStyle(DialogFragment.STYLE_NORMAL, android.R.style.Theme_Holo_Light_NoActionBar);
+    }
+
+    @Override
+    public void onActivityCreated(Bundle savedInstanceState)
+    {
+        super.onActivityCreated(savedInstanceState);
+    }
+
+    @Override
+    public View onCreateView(final LayoutInflater inflater, final ViewGroup container,
+                             Bundle savedInstanceState) {
+
+        ViewGroup dialogView = (ViewGroup) inflater.inflate(R.layout.fragment_add_member, container, false);
+
+
+
+        return dialogView;
+    }
+
+    @Override
+    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        view.findViewById(R.id.close).setOnClickListener(this);
+        view.findViewById(R.id.layout_add_child_under_five).setOnClickListener(this);
+        view.findViewById(R.id.layout_add_other_family_member).setOnClickListener(this);
+    }
+
+    @Override
+    public void onAttach(Activity activity) {
+        super.onAttach(activity);
+        // Verify that the host activity implements the callback interface
+//        try {
+//            // Instantiate the WeightActionListener so we can send events to the host
+//            listener = (WeightActionListener) activity;
+//        } catch (ClassCastException e) {
+//            // The activity doesn't implement the interface, throw exception
+//            throw new ClassCastException(activity.toString()
+//                    + " must implement WeightActionListener");
+//        }
+    }
+
+
+
+    @Override
+    public void onStart() {
+        super.onStart();
+        // without a handler, the window sizes itself correctly
+        // but the keyboard does not show up
+        new Handler().post(new Runnable() {
+            @Override
+            public void run() {
+                getDialog().getWindow().setLayout(FrameLayout.LayoutParams.MATCH_PARENT, FrameLayout.LayoutParams.MATCH_PARENT);
+
+            }
+        });
+
+    }
+
+
+
+    public static AddMemberFragment newInstance() {
+        AddMemberFragment addMemberFragment = new AddMemberFragment();
+        return addMemberFragment;
+    }
+
+
+    @Override
+    public void onClick(View v) {
+        switch (v.getId()) {
+            case R.id.close:
+                dismiss();
+                break;
+            case  R.id.layout_add_child_under_five:
+                try {
+                    startForm(Constants.JSON_FORM.CHILD_REGISTER,"","","",familyBaseEntityId);
+                } catch (Exception e) {
+                    e.printStackTrace();
+                }
```

Comment:
Add a break; Currently adding child under 5 is opening both the Add Child and Family register

Reply:
..Regarding "search name or ID">"Find name od ID" yes i'll commit this with separate issue #12
..Yeah miss break in case child add.adding now